tests/unit/amo/api/test_reviews.js

```diff
@@ -123,37 +134,35 @@ describe('amo.api', () => {
           auth: true,
           state: params.apiState,
         })
-        .returns(Promise.resolve(genericApiResponse));
+        .returns(Promise.resolve(submitReviewResponse()));
 
-      return submitReview(params).then(() => {
-        mockApi.verify();
-      });
+      await submitReview(params);
+      mockApi.verify();
     });
   });
 
   describe('getReviews', () => {
-    it('allows you to fetch reviews by any param', () => {
+    it('allows you to fetch reviews by any param', async () => {
```

Comment:
Sorry I didn't catch this before but I'm a bit confused by this spec. Is it just saying it can fetch the reviews by addon ID instead of slug?

Reply:
Hmm, it's old code so I'm not sure. I think "it allows you to fetch reviews with filters" will be better.